src/main/java/uk/ac/ebi/embl/api/validation/submission/SubmissionValidationPlan.java

```diff
@@ -317,32 +362,40 @@ private void validateAnnotationOnlyFlatfile() throws ValidationEngineException
 
 				if(sequenceDB!=null)
 					check.setSequenceDB(sequenceDB);
-				if(!check.check(annotationOnlyFlatfile))
-					throwValidationCheckException(FileType.ANNOTATION_ONLY_FLATFILE, annotationOnlyFlatfile);
+				result = check.check(annotationOnlyFlatfile);
+				if(!result.isValid()) {
+					result.setDefaultOrigin(new DefaultOrigin(getValidationErrorMessage(FileType.ANNOTATION_ONLY_FLATFILE,annotationOnlyFlatfile, check)));
+					return result;
+				}
 			}
 		}catch(ValidationEngineException e)
 		{
 			throwValidationEngineException(FileType.ANNOTATION_ONLY_FLATFILE,e,fileName);
 		}
```

Comment:
Now sure why we don't just do:

catch(Exception e)
{
throwValidationEngineException(FileType. ANNOTATION_ONLY_FLATFILE,e,"master.dat");
}